apps/web/src/services/dashboards/components/DashboardWidgetMoreOptions.vue

```diff
@@ -27,6 +27,16 @@ const state = reactive({
         const properties = widgetFormGetters.widgetConfig?.options_schema?.properties ?? {};
         return Object.entries(properties);
     }),
+    widgetOptionMenuItems: computed<MenuItem[]>(() => {
+        const _refinedMenu = refinedMenu.value.map((d) => {
+            const schemaProperty = widgetFormGetters.widgetConfig?.options_schema?.properties?.[d.name];
+            if (schemaProperty?.fixed || schemaProperty?.readonly) {
+                return { ...d, disabled: true };
+            }
+            return d;
+        });
+        return _refinedMenu;
+    }),
```

Comment:
description: disable when it's readonly or fixed option

![스크린샷 2024-01-12 오후 3 08 58](https://github.com/cloudforet-io/console/assets/18563857/7b6940d5-f00c-4ed0-94f5-0f4454bd3048)